src/IAM/Domain/Auth/Exception/BadRequestException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\Domain\Auth\Exception;
+
+final class BadRequestException extends \RuntimeException
```

Comment:
```suggestion
final class BadRequestException extends RuntimeException
```

Simplify FQNs. For reference, see [afe10ce](https://github.com/CPAprince/Twitter/pull/20/changes/afe10ce7843d0cf1e03526e247450f10540d6159#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0)